### DISCUSSION TITLE
Error with Function Tool Calling

### DISCUSSION THREAD

**Discussion by Sim2K** [FlowiseAI/Flowise#1517] [General]
Ive just started getting Assistant errors on chats I was NOT getting errors on before even though I have made no changes. New Assistants work but not with calling a Tool. I get the following error:

`Error: Error submitting tool outputs: requires_action, Thread ID: thread_mZqhkk3svprl0DadCdSdVMGd, Run ID: run_PLGZTmBfLH8nAxKe9vobYVmj`

Anyone else getting this?

Had to go back to the old way.

**Comment by acoffin76** [comment]
I'm getting this error, as well. I just upgrade to the latest version.

[2025-03-04 01:53:43 +0000] [3691752] [ERROR] Call to LLM failed: {'statusCode':
 500, 'success': False, 'message': 'Error: predictionsServices.buildChatflow - E
rror: Error submitting tool outputs: requires_action, Thread ID: thread_cRB4fYpQ
2IYwhcb2KBuJSAoL, Run ID: run_EJ4jZG5fHYBZcqjusJ3ybcRD', 'stack': {}}

**Comment by tmowes** [comment]
i'm using assistant to call multiple tools but sometimes random i getting this error: predictionsServices.buildChatflow - Error: Error submitting tool outputs: requires_action Thread ID: .... Run ID: ...

there is something i missing?
